client/src/components/v2/OverlayMessage.js

```diff
@@ -0,0 +1,47 @@
+import React, { useCallback } from 'react';
+import { Button, Card, FormControl, InputGroup } from 'react-bootstrap';
+import { useMessageState } from 'api/v2/camera';
+
+/**
+ * Text field for sending messages to the rider via the active overlay.
+ *
+ * @returns {React.Component} Component
+ */
+export default function OverlayMessage() {
+  const { message, setMessage, sendMessage } = useMessageState();
+
+  const handleMessageChange = useCallback(
+    (event) => {
+      setMessage(event.target.value);
```

Comment:
Store this in a `useState` for this component (temporarily) and then when user clicks the Send button, we run `sendMessage(message)`.